app/views/curation_concerns/base/_member.html.erb

```diff
@@ -0,0 +1,16 @@
+<tr class="<%= dom_class(member) %> attributes">
+  <td class="thumbnail">
+    <% if member.reader? %>
```

Comment:
@jcoyne is the downloads controller used in this partial when `render_thumbnail_tag` is called?